packages/ethereum/deploy/20_mintDevNftTransferOwnership.ts

```diff
@@ -110,24 +109,29 @@ const main: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     const registryProxy = (await ethers.getContractFactory('HoprStakingProxyForNetworkRegistry')).attach(
       registryProxyDeployment.address
     ) as HoprStakingProxyForNetworkRegistry
-    await registryProxy.ownerBatchAddNftTypeAndRank([devNftIndex], [DEV_NFT_BOOST])
+    const ownerNftTx = await registryProxy.ownerBatchAddNftTypeAndRank([devNftIndex], [DEV_NFT_BOOST])
+    // don't wait when using local hardhat because its using auto-mine
+    if (!environment.match('hardhat')) {
+      await ethers.provider.waitForTransaction(ownerNftTx.hash, 2)
+    }
 
     try {
       // mint minimum stake to addresses that will stake and are binded to nodes in NR
-      const tokenContract = await deployments.get('HoprToken')
-      const hoprToken = (await ethers.getContractFactory('HoprToken')).attach(tokenContract.address) as HoprToken
-      await hoprToken.mint(
-        CLUSTER_NETWORK_REGISTERY_LINKED_ADDRESSES[0],
-        MIN_STAKE,
-        ethers.constants.HashZero,
-        ethers.constants.HashZero
-      )
-      await hoprToken.mint(
-        CLUSTER_NETWORK_REGISTERY_LINKED_ADDRESSES[2],
-        MIN_STAKE,
-        ethers.constants.HashZero,
-        ethers.constants.HashZero
-      )
+      const tokenContract = await deployments.get('xHoprMock')
+      const hoprToken = (await ethers.getContractFactory('ERC677Mock')).attach(tokenContract.address) as ERC677Mock
```

Comment:
@QYuQianchen I guess this is highly specific for xDAI? Probably worth to add a small comment